### PR TITLE
Es6 features practise

### PR DESCRIPTION
**Pull request by xydoy92**

### CODE REVIEW

File: es6-features-practise/test/iterators.js [hunk 1]
```diff
@@ -122,23 +129,23 @@ describe('A simple iterable without items inside, implementing the right protoco
   });
   describe('using the iterable', function() {
     it('it contains no values', function() {
-      let values;
+      let values = 0;
```
Comment (tr3v3r) [inline es6-features-practise/test/iterators.js:132]: `let values = ''`

in your case, `values` will be `0`

File: es6-features-practise/test/templateStrings.js [hunk 2]
```diff
@@ -133,28 +141,28 @@ describe('Tagged template strings, are an advanced form of template strings', fu
...
     function firstChar(strings) {
-      return strings;
+      return strings.raw;
     }
     assert.equal(firstChar`\n`, '\\n');
   });
   it('`raw` can access the backslash of a line-break', function() {
     function firstCharEntered(strings) {
-      var lineBreak = strings.raw;
+      var lineBreak = strings.raw.toString()[0];
```
Comment (tr3v3r) [inline es6-features-practise/test/templateStrings.js:150]: `strings.raw[0];`

**Review by tr3v3r — APPROVED** [review]

File: es6-features-practise/test/templateStrings.js (hunk 2 above)
Comment (xydoy92) [reply es6-features-practise/test/templateStrings.js:150]: No, `strings.raw[0]` gave an error by `npm test`.

File: es6-features-practise/test/iterators.js (hunk 1 above)
Comment (xydoy92) [reply es6-features-practise/test/iterators.js:132]: Why is this better?

**Pull request merged**